spec/unit/storage_spec.rb

```diff
@@ -3,6 +3,42 @@
 describe AssetSync::Storage do
   include_context "mock Rails without_yml"
 
+  describe '#local_files' do
+    context 'default' do
+      let(:config) { AssetSync::Config.new }
+      let(:storage) { AssetSync::Storage.new(config) }
+
+      it 'returns directories and files inside public/assets' do
+        expect(storage.local_files).to eq([
+          'assets/javascripts',
+          'assets/javascripts/application.js',
+        ])
+      end
+    end
+
+    context 'with include_webpacker_assets = true' do
+      let(:config) do
+        config = AssetSync::Config.new
+        config.include_webpacker_assets = true
+        config
+      end
+
+      let(:storage) { AssetSync::Storage.new(config) }
+
+      module Webpacker
+        class Configuration
+          def self.paths
+            {}
+          end
+        end
+      end
```

Comment:
This could be dangerous. Could we use RSpec's `stub_const` instead? https://relishapp.com/rspec/rspec-mocks/v/2-11/docs/stubbing-constants/stub-defined-constant#stub-nested-constant